### PR TITLE
Add autofix support for CSS with standard syntax

### PR DESCRIPTION
**Pull request by gucong3000**
> Which issue, if any, is this issue related to?

no

> Is there anything in the PR that needs further explanation?

Autofix css standard syntax error by `postcss-safe-parser`.

### CODE REVIEW

**Comment by jeddy3** [comment]
@gucong3000 Thanks! 

As per [the guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/issues.md), only use `type:` labels for issues. The idea is that you should _create an issue to discuss a feature first_, before submitting a PR. Those issues are then assigned `type:` labels and PRs should reference the original issue.

File: package.json [hunk 1]
```diff
@@ -85,15 +86,13 @@
     "eslint-config-stylelint": "^7.0.0",
     "file-exists-promise": "^1.0.2",
     "flow-bin": "^0.54.0",
-    "fs-promise": "^2.0.3",
```
Comment (ntwb) [inline package.json]: I know this package has been deprecated in favour of `fs-extra` per https://www.npmjs.com/package/fs-promise but the `pify` package has not been added via npm install pify --save-dev` to replace `fs-promise` here in `package.json`

File: lib/__tests__/fixtures/syntax_error.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+a {
```
Comment (ntwb) [inline lib/__tests__/fixtures/syntax_error.scss]: There should be a _end-of-file_ new line here.

File: package.json (hunk 1 above)
Comment (jeddy3) [reply package.json]: I believe `pify` is already available [as a dependency](https://github.com/stylelint/stylelint/blob/90149a5e650e1adb138206b29e055847da59f4ba/package.json#L60)

Comment (ntwb) [reply package.json]: That it is, my bad, sorry 😏

**Comment by jeddy3** [comment]
@gucong3000 Sorry about the delay merging this in. I've only had time to triage issues and not look into PRs. With 3 approvals, this can be merged by any of the team :)

**Comment by jeddy3** [comment]
> -   Added: autofix of syntax errors in standard CSS e.g. unclosed braces and brackets ([#2886](https://github.com/stylelint/stylelint/issues/2886)).

**Pull request merged**